libs/features/georoutingv2/georoutingv2.js

```diff
@@ -35,35 +35,25 @@ export const getCookie = (name) => document.cookie
   .find((row) => row.startsWith(`${name}=`))
   ?.split('=')[1];
 
-/* c8 ignore next 16 */
-const geo2jsonp = (callback) => {
-  // Setup a unique name that can be called & destroyed
-  const callbackName = `jsonp_${Math.round(100000 * Math.random())}`;
-
-  const script = document.createElement('script');
-  script.src = `https://geo2.adobe.com/json/?callback=${callbackName}`;
-
-  // Define the function that the script will call
-  window[callbackName] = (data) => {
-    delete window[callbackName];
-    document.body.removeChild(script);
-    callback(data);
-  };
-
-  document.body.appendChild(script);
-};
-
-const getAkamaiCode = () => new Promise((resolve) => {
+const getAkamaiCode = () => new Promise((resolve, reject) => {
   const urlParams = new URLSearchParams(window.location.search);
   const akamaiLocale = urlParams.get('akamaiLocale') || sessionStorage.getItem('akamai');
   if (akamaiLocale !== null) {
     resolve(akamaiLocale.toLowerCase());
   } else {
     /* c8 ignore next 5 */
-    geo2jsonp((data) => {
-      const code = data.country.toLowerCase();
-      sessionStorage.setItem('akamai', code);
-      resolve(code);
+    fetch('https://geo2.adobe.com/json/').then((resp) => {
```

Comment:
Why not keep the `geo2jsonp` function instead of inlining it?

I thought that the randomized callback name is also for cache busting?